src/client/pages/StaffViewConfirmationPage/index.js

```diff
@@ -0,0 +1,165 @@
+
+  function handlePrint() {
+    window.print();
+  }
+
+  function CertificationStatus() {
+    switch (status) {
+      case statuses.NOT_STARTED:
+        return (
+          <React.Fragment>
+            <span className="badge not-started">
+              {t("staff-view-confirmation.not-started.status")}
+            </span>
+            {/* TODO(kalvin): update with EDD content when received */}
+            <p>{t("staff-view-confirmation.not-started.p1")}</p>
+          </React.Fragment>
+        );
+      case statuses.IN_PROGRESS:
+        return (
+          <React.Fragment>
+            <span className="badge in-progress">
+              {t("staff-view-confirmation.in-progress.status")}
+            </span>
+            {/* TODO(kalvin): update with EDD content when received */}
+            <p>{t("staff-view-confirmation.in-progress.p1")}</p>
+          </React.Fragment>
+        );
+      case statuses.COMPLETED:
+        return (
+          <React.Fragment>
+            <span className="badge completed">
+              {t("staff-view-confirmation.completed.status")}
+            </span>
+            {/* TODO(kalvin): make last name real */}
+            <p>
+              {t("staff-view-confirmation.completed.last-name")}
+              <br />
+              {t("staff-view-confirmation.completed.confirmation-number") +
+                shortConfirmationNumber}
+            </p>
+            <p>
+              <Trans
+                t={t}
+                i18nKey="staff-view-confirmation.completed.p1"
+                values={{ confirmationNumber, shortConfirmationNumber }}
+              >
+                Note: The claimant may have seen or saved a much longer
+                confirmation number in the past. If so, it was{" "}
+                <strong>adfdsaf</strong>. If they log in now, they will see
+                <strong>adfdsafdfds</strong> instead, which is the last 7
+                characters of the original number.
+              </Trans>
+            </p>
+          </React.Fragment>
+        );
+    }
+  }
+
+  function ClaimantContent() {
+    return status === statuses.COMPLETED ? (
+      <div className="subtle-blockquote mb-5">
+        <p>{t("retrocerts-confirmation.p1b")}</p>
+        <Trans t={t} i18nKey="retrocerts-confirmation.p2">
+          If you are still unemployed, continue to certify for benefits every
+          two weeks. The fastest way is to use{" "}
+          <a href={t("links.edd-login")}>UI Online</a>.
+        </Trans>
+      </div>
+    ) : (
+      <div className="subtle-blockquote mb-5">
+        <p>{t("retrocerts-weeks.p3")}</p>
+        <p>{t("retrocerts-weeks.p4")}</p>
+      </div>
+    );
+  }
+
+  return (
+    <div id="overflow-wrapper">
+      <Header />
+      <main id="certification-page" className="pb-5">
+        <div className="container p-4">
+          <h1>{t("staff-view-confirmation.title")}</h1>
+          <h3>{t("staff-view-confirmation.header1")}</h3>
```

Comment:
This is an accessibility issue. The headers have to go in order from 1-6. Bootstrap has helper classes that let you style h2s like h3s though.

Reply:
oh interesting, ok. thanks!